src/detect-engine.c

```diff
@@ -2523,6 +2523,9 @@ static DetectEngineCtx *DetectEngineCtxInitReal(
         goto error;
     }
 
+    // TODO test enable flowbits prefilter
```

Comment:
Still need to check how we may enable a selection of prefilter engines for keywords, but not all.